src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageCopyHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+{
+    public class PackageCopyHandler : IPackageCopyHandler
```

Comment:
>IPackageCopyHandler [](http://example.com/codeflow?start=38&length=19)

can we remove this interface and make CopyPackageFile virtual?

Reply:
As recommended in another comment, I have moved the `CopyPackageFile` method to `PackageDownloadHandler`.